onnx/backend/test/loader/__init__.py

```diff
@@ -52,11 +52,17 @@ def load_node_tests(data_dir=os.path.join(DATA_DIR, 'node')):
     return testcases
 
 
-def _load_model_tests(data_dir):
+
+def load_model_tests(data_dir=os.path.join(DATA_DIR, 'model'), kind=None):
```

Comment:
We have load_model_tests, load_node_tests, collect_tests (in both case.model and case.node). Some comments can help us remember the purpose of the functions.

Reply:
Added